<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_container" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionRotate"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu3D">
    <property name="title">
     <string>3D</string>
    </property>
    <addaction name="action3D_Preview"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu3D"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRotate"/>
   <addaction name="actionDelete"/>
   <addaction name="action3D_Preview"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/pen.png</normaloff>:/assets/icons/png/pen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/diskette.png</normaloff>:/assets/icons/png/diskette.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/direction.png</normaloff>:/assets/icons/png/direction.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/technology.png</normaloff>:/assets/icons/png/technology.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/reload-webpage.png</normaloff>:/assets/icons/png/reload-webpage.png</iconset>
   </property>
   <property name="text">
    <string>rotate</string>
   </property>
   <property name="toolTip">
    <string>rotates the selected component</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/cross-sign.png</normaloff>:/assets/icons/png/cross-sign.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>deletes the selected component</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action3D_Preview">
   <property name="icon">
    <iconset resource="levelbuilder.qrc">
     <normaloff>:/assets/icons/png/direction.png</normaloff>:/assets/icons/png/direction.png</iconset>
   </property>
   <property name="text">
    <string>3D-Preview</string>
   </property>
   <property name="toolTip">
    <string>Show in 3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="levelbuilder.qrc"/>
 </resources>
 <connections/>
</ui>
